src/utils/index.ts

```diff
@@ -95,3 +95,13 @@ export function isInvalidEmail(email: string) {
 
     return !EMAIL_RE.test(email);
 }
+
+export function isContributor(contributor: unknown): contributor is Contributor {
+    if (!contributor) {
+        return false;
+    }
```

Comment:
I suppose it's better to add check for object here:

```ts
    if (!contributor || typeof contributor !== 'object') {
        return false;
    }
```

Reply:
fixed